### PR TITLE
Use correct filename for pnpm workspace definition and add glossary terms for Volto

### PR DESCRIPTION
**Pull request by stevepiercy**

### CODE REVIEW

**Comment by stevepiercy** [comment]
For this round, I'll defer merging until I get an approval. There may be other changes that need to happen as part of my review of https://github.com/plone/volto/pull/5882

**Review by ichim-david — DISMISSED** [review]

**Pull request merged**
